Assets/scss/components/_project-leads.scss

```diff
@@ -0,0 +1,85 @@
+.project-leads-component {
+    margin: toRem(20);
+    padding: toRem(20);
+    border: toRem(1) solid var(--very-pale-shade-gray);
+    border-radius: toRem(8);
+    box-shadow: 0 toRem(2) toRem(4) rgba(0, 0, 0, 0.1);
+
+    h2 {
+        font-size: toRem(24);
+        margin-block-end: toRem(10);
+        color: var(--black);
+    }
+
+    .experts {
+        display: grid;
+        grid-template-columns: repeat(5, 1fr);
+        grid-gap: toRem(20);
+        border-top: 1px solid var(--black);
+
+        @include max-screen(md) {
```

Comment:
@dkdeepesh  add a container width  like 1177px or converted it into rem.

Cc:-  @msharmaRP @jvaishnavrp